server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/oms/client/BaseOmsClient.java

```diff
@@ -100,6 +100,7 @@ private <T> OmsApiReturnResult<T> resolveResponseEntity(ClientRequestParams requ
                     : ErrorCodes.BadRequest;
             throw new OmsException(errorCode, responseEntity.toString(), null, responseEntity.getStatusCode());
         }
+        log.info("process oms request [{}] with response [{}]", requestParams, responseEntity.getBody());
```

Comment:
odc use k=v style for log output,
such as log.info("process oms request, request={}, response={}", xxx, xxx)

Reply:
corrected